packages/manager/modules/hub/src/components/carousel/carousel.controller.js

```diff
@@ -0,0 +1,57 @@
+import map from 'lodash/map';
+import first from 'lodash/first';
+import set from 'lodash/set';
+import find from 'lodash/find';
+import findIndex from 'lodash/findIndex';
+import filter from 'lodash/filter';
+
+import { Environment } from '@ovh-ux/manager-config';
+
+export default class {
+  /* @ngInject */
+  constructor($window, OvhApiNotificationAapi, TranslateService) {
+    this.$window = $window;
+    this.OvhApiNotificationAapi = OvhApiNotificationAapi;
+    this.TranslateService = TranslateService;
+    this.REGION = Environment.getRegion();
+  }
+
+  $onInit() {
+    return this.OvhApiNotificationAapi.query(
+      this.TranslateService.getUserLocale(),
+      this.REGION,
+    )
+      .$promise.then((notificationList) => {
+        this.notifications = map(
+          filter(
+            notificationList,
+            (notification) =>
+              notification.level === 'warning' ||
+              notification.level === 'error',
+          ),
+          (notification) => ({
+            ...notification,
+            active: false,
+          }),
+        );
+        set(first(this.notifications), 'active', true);
+      })
+      .catch(() => undefined);
```

Comment:
seems useless

Reply:
notifications will be passed to the component, you don't need to fetch for them in component